File: javascript/pares_consecutivos.js
let x, soma;

x = parseInt(prompt("Digite um numero inteiro"));

while (x != 0) {
    if (x % 2 == 0) {
        soma = 5 * x + 20;
    } else {
        soma = 5 * x + 25;
    }

    console.log("SOMA = %d", soma);
    x = parseInt(prompt("Digite um numero inteiro"));
}

File: python/pares_consecutivos.py
x = int(input("Digite um numero inteiro: "))

while x != 0:
    soma = 0

    if x % 2 != 0:
        x += 1

    for i in range(x, x + 9, 2):
        soma += i

    print(f"SOMA =", soma)
    x = int(input("Digite um numero inteiro: "))
